.github/workflows/config.yml

```diff
@@ -52,22 +52,64 @@ jobs:
           path: resources/buildstock.csv
           name: samples
 
-      - name: Upload integration results
-        uses: actions/upload-artifact@v2
-        with:
-          path: test/test_samples_osw/baseline/results
-          name: results
-
       - name: Upload schedules
         uses: actions/upload-artifact@v2
         with:
           path: resources/measures/ResidentialScheduleGenerator/tests/output/test_sweep_building_ids_and_num_occupants
           name: schedules
 
+  bsb:
+    runs-on: ubuntu-latest
+    needs: build
+    steps:
+      - name: Show environment
+        run: |
+          ruby -v
+          python -v
+
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.head_ref }}
+
+      - name: Run buildstockbatch
+        run: |
+          pip install git+https://github.com/NREL/buildstockbatch.git@develop
+          
+          buildstock_docker project_testing/testing_baseline.yml > project_testing/testing_baseline.txt
+          buildstock_docker project_national/national_baseline.yml > project_national/national_baseline.txt
+          
+          buildstock_docker project_testing/testing_upgrades.yml > project_testing/testing_upgrades.txt
+          buildstock_docker project_national/national_upgrades.yml > project_national/national_upgrades.txt
```

Comment:
Is there a reason you're suppressing direct output here and requiring someone to go to the CI artifacts? And what if this job fails and then you can't retrieve the artifacts? Just seems less convenient.

Reply:
Yes. I was getting "exit code 1 something something" when not redirecting output to a text file. I mentioned to @nmerket, and it was news to him. 🤷